lib/content-services/src/lib/dialogs/download-zip/download-zip.dialog.ts

```diff
@@ -79,7 +81,12 @@ export class DownloadZipDialogComponent implements OnInit {
 
         this.downloadZipService.getDownload(downloadId).subscribe((downloadEntry) => {
             if (downloadEntry.entry) {
-                if (downloadEntry.entry.status === 'DONE') {
+                if (downloadEntry.entry.status === FileDownloadStatus.IN_PROGRESS) {
```

Comment:
this is not going to work, as the `getDownload` is a one-time REST api call, wrapped with the observable. Have you seen it working? Can you record a video, thanks

Reply:
excellent, thanks